Audioのインターフェースを明示的に実装

kmamonnassi opened the pull request:
public void DoMethod →IInterface.DoMethod　のように明示的に書くように

kmamonnassi committed:
タイトルが完成

kmamonnassi committed:
Merge branch 'main' into title

kmamonnassi committed:
コメントアウトを追加
AnimatedButtonのフォルダ構成を変更

kmamonnassi committed:
コメントアウトを追加その2

kmamonnassi committed:
音量設定、画面設定、設定ファイルの保存システムが完成

kmamonnassi committed:
asmdefに不備があったので修正

kmamonnassi committed:
オーディオ関係のファイルを削除

kmamonnassi committed:
オーディオの設定を設定ファイルから参照できるように

kmamonnassi committed:
SE、BGMを流せるように

Commit:
プレイヤーモデル追加
歩きモーション一旦完成

kmamonnassi committed:
鳴らしたSEを停止、再生できるように
BGMがフェードアウト中にキューシートが削除されないように

kmamonnassi committed:
model、presenter、viewなどに切り離されていたasmdefを1つにまとめた

kmamonnassi committed:
model、presenter、viewと分けていたasmdefを一つにまとめた

kmamonnassi committed:
Merge branch 'settings_model' into audio

kmamonnassi committed:
asmdefの情報を修正

kmamonnassi committed:
コメントアウトを追加

Commit:
歩きアニメーションver2
モデル更新

Commit:
Merge branch 'audio' into feature/Motion_Player

# Conflicts:
#	RoboPro/UserSettings/Layouts/default-2021.dwlt

kmamonnassi committed:
インターフェースを明示的に実装

kmamonnassi committed:
インターフェースを明示的に実装

Commit:
Player_Access_Start(アクセス開始)
Player_Access_End(アクセス終了)
Player_Idle(待機アニメーション)

Commit:
すみませんdevelopで作業してました許して下さいなんでもしますから
複数のアニメーションとアニメーター、確認用のプレイヤー操作スクリプトを作成

Commit:
テスト用プログラムの修正
アニメーターの修正

nkc20206086 committed:
Merge pull request #12 from nkc20206086/feature/Motion_Player

other:Player_Animation

Commit:
ふらつきアニメーション

nkc20206086 committed:
Merge pull request #13 from nkc20206086/feature/feature_PlayerAnimation_Unsteady

other:Animation_Player_Unsteady

komugikoShimizu committed:
コマンド入れ替え処理

komugikoShimizu committed:
Merge branch 'develop' into Programming

# Conflicts:
#	RoboPro/UserSettings/Layouts/default-2021.dwlt

kmamonnassi committed:
AudioPlayBackをAudioPlaybackに変更
インターフェースを明示的に実装
コメントアウトの追加など

kmamonnassi committed:
Merge branch 'develop' into audio

# Conflicts:
#	RoboPro/UserSettings/Layouts/default-2021.dwlt

kmamonnassi committed:
SceneAudioImporterと、そのEditorのコメントアウトを追加

kmamonnassi committed:
CueSheetTypeにコメントアウトを追加

amakasuzirofight commented on RoboPro/Assets/VertToon/Editor/CustomUIParts/Vec2Drawer.cs:
ここの処理はさっぱりだけどマジックナンバーを見かけたので一応。
なんか意図があったら言ってね

amakasuzirofight reviewed (changes requested):
コードレビューしたよ

kmamonnassi commented on RoboPro/Assets/Scripts/Settings/Model/Audio/AudioSettings.cs:
Funcを使わないでも良いように作り直してみた

komugikoShimizu committed:
コードレビュー、セーブ結果反映

komugikoShimizu committed:
コードレビュー反映ののち、セーブのシステムを書き換えた

ayaha401 committed:
pro:edit VertToonUpdate
v.0.1.1に更新

ayaha401 committed:
Merge pull request #18 from nkc20206086/feature/VertToonUpdate_V0-1-1

pro:edit VertToonUpdate

ayaha401 committed:
pro:add Billboard
UIを常にカメラの方向を向かせるスクリプトを作成

ayaha401 committed:
pro:edit UICamera
UIをオブジェクトに対して優先描画するためにUI用のCameraを作った

amakasuzirofight committed:
Merge pull request #20 from nkc20206086/feature/Test_BillBoardUI

UIを常にカメラの方向を向かせるスクリプトを作成

ayaha401 committed:
pro:add MVPModule
OSSをインストール

ayaha401 committed:
Merge pull request #21 from nkc20206086/feature/AddMVPModule

pro:add MVPModule

ayaha401 committed:
pro:add interactUI
インタラクトを教えるUIを作成

ayaha401 committed:
pro:Edit InteractUIController
UIを指定位置に移動させて表示するように変更した

ayaha401 committed:
pro:del TestInteractUI
テスト用のスクリプトを削除

pro:add interface
インターフェースで実装

ayaha401 committed:
pro:edit InteractUIController
不要なusingを消去

ayaha401 committed:
pro:add HowToOperate
2秒間入力が無かったらUIを表示するプログラムを作った

ayaha401 committed:
pro:edit HowToOperate
UniTaskで実装したパターンを作った

amakasuzirofight committed:
Merge pull request #23 from nkc20206086/feature/HowToOperate

pro:add HowToOperate

Commit:
other:add UIprefab
UIのPrefab、Fontを追加しました。

nkc20206086 committed:
Merge pull request #25 from nkc20206086/feature/addUIAsset

other:add UIprefab

komugikoShimizu committed:
Merge pull request #17 from nkc20206086/Programming

コードレビューの反映、セーブ機能の変更

nkc-TakaoOnda committed:
other:add Cinemachine
Cinemachineをインストールしました。

nkc-TakaoOnda committed:
Merge pull request #26 from nkc20206086/feature/feature_Camera

other:add Cinemachine

nkc-TakaoOnda committed:
other:add Install InputSystem
InputSystemをインストールした。
PlayerSetting→Player→Active Input Handling→Bothに変更
(今までのInputも使用できる状態になってます)

nkc-TakaoOnda committed:
Merge pull request #27 from nkc20206086/feature/feature_Install_InputSystem

other:add Install InputSystem

nkc-TakaoOnda committed:
pro:add MainCamera_Move&Camera_Vector_Getter
ゲーム中のカメラの動きとカメラの角度を取得できるようにした

nkc-TakaoOnda committed:
pro:add MainCamera_Move & MainCameraVectorGetter
メインカメラのキーとマウスで動かすことができるようになったのと、メインカメラの向いている方向に対して補正値を計算・渡せるようにした

nkc-TakaoOnda committed:
Merge pull request #29 from nkc20206086/feature/feature_MainGame_Camera

Feature/feature main game camera

Commit:
other:add BackgroundObjects
木や草、石等の背景オブジェクトを作成して入れた

SoseiNKC13 committed:
Merge pull request #30 from nkc20206086/feature/feature_BGObject

other:add BackgroundObjects

Commit:
other:add Player_Prefab
プレイヤーのプレハブを作った

SoseiNKC13 committed:
Merge pull request #32 from nkc20206086/feature/feature_BGObject3

other:add Player_Prefab

Commit:
other:add Player_Prefab
直前のプルリクエストでは「PlayerAnimationController」をプレハブ化したが、今回はその親オブジェクトにあたる「Player」をプレハブ化

SoseiNKC13 committed:
Merge pull request #33 from nkc20206086/feature/feature_BGObject3

other:add Player_Prefab

nkc-TakaoOnda committed:
pro:add StateMachine
プレイヤーの状態遷移を作成(デバッグログ表示)

nkc-TakaoOnda committed:
Merge pull request #34 from nkc20206086/feature/feature_Player_Statemachine

pro:add StateMachine

nkc-TakaoOnda committed:
pro:add Player_Move
プレイヤーが移動できるようになった
移動中にMoveアニメーションが出るようになった

nkc-TakaoOnda committed:
Merge pull request #35 from nkc20206086/feature/feature_Player_Move

pro:add Player_Move

nkc-TakaoOnda committed:
pro:add PlayerDizzy
ふらつきが途中だけどプッシュ命令があったのでする

nkc-TakaoOnda committed:
Merge pull request #36 from nkc20206086/feature/feature_Player_Dizzy

pro:add PlayerDizzy

Commit:
other:Add AccessPoint
アクセスポイントのプレハブを作成
それに伴い色のマテリアルも作成
その他細かい変更やフォルダの追加もあるが割愛

SoseiNKC13 committed:
Merge pull request #37 from nkc20206086/feature/feature_BGObject3

other:add AccessPoint

amakasuzirofight committed:
向き補正のバグを無くした

amakasuzirofight committed:
Merge branch 'feature_bugFix' into develop

kmamonnassi committed:
asmdefを切りすぎていたので削除

kmamonnassi committed:
Merge branch 'develop' into title

# Conflicts:
#	RoboPro/UserSettings/EditorUserSettings.asset

ayaha401 committed:
Merge branch 'develop' into feature/InteractUI

ayaha401 committed:
pro:edit Enumの名前修正

amakasuzirofight committed:
Merge pull request #42 from nkc20206086/title

Title作成

amakasuzirofight committed:
Merge pull request #22 from nkc20206086/feature/InteractUI

インタラクト用のUIを作成

kmamonnassi committed:
Merge branch 'develop' into audio

# Conflicts:
#	RoboPro/UserSettings/Layouts/default-2021.dwlt
